HandleNHDeleteInstallationCallActivity/index.ts

```diff
@@ -0,0 +1,24 @@
+﻿import { createActivity } from "../utils/durable/activities";
+import { buildNHService } from "../utils/notificationhubServicePartition";
+import {
+  ActivityBodyImpl,
+  ActivityInput,
+  ActivityResultSuccess,
+  getActivityBody
+} from "./handler";
+
+export {
+  ActivityBodyImpl,
+  ActivityInput,
+  ActivityResultSuccess
+} from "./handler";
+export const activityName = "HandleNHDeleteInstallationCallActivity";
```

Comment:
Question: I wonder if we could just infer this from the actual file, inspecting `__dirname` variable.

Reply:
I was asking myself the same question.. we did 30, why not go for 31? Just.. can we plan it for the next iterations? I'll do some tests offline

Reply:
Yes, next iterations for sure. We may want to converge to a common structure for activities and orchestrators first, then we can write some helpers (or some generated code)